### ISSUE TITLE
invalid prometheus metrics output in proxy

### ISSUE THREAD

**Issue by siggy** [runconduit/conduit#600]
The `/metrics` endpoint on `conduit-proxy` contains invalid output in the form of repeated commas, preventing Prometheus collection:
```
direction="outbound",,status_code="200"
```
## steps to reproduce

```bash
$ git checkout siggy/prom-scrape
$ DOCKER_TRACE=1 bin/docker-build
$ bin/conduit install | kubectl apply -f -
$ curl https://raw.githubusercontent.com/runconduit/conduit-examples/master/emojivoto/emojivoto.yml | bin/conduit inject - | kubectl apply -f -
$ kubectl -n emojivoto exec -it $(kubectl --namespace=emojivoto get po --selector=app=vote-bot -o jsonpath='{.items[*].metadata.name}') -c conduit-proxy curl localhost:4191/metrics
...
response_total{authority="web-svc.emojivoto:80",direction="outbound",,status_code="200"} 2868
response_total{authority="web-svc.emojivoto:80",direction="outbound",,status_code="500"} 334
```

## further validation

We can also validate the issue via the prometheus targets page:

```
$ kubectl -n conduit port-forward $(kubectl --namespace=conduit get po --selector=conduit.io/control-plane-component=prometheus -o jsonpath='{.items[*].metadata.name}') 9090:9090
$ open http://localhost:9090/targets
```

![screen shot 2018-03-21 at 5 51 20 pm](https://user-images.githubusercontent.com/236915/37745152-890d9528-2d30-11e8-815c-61db6b376a6e.png)

relates to #569, #583

**Comment by hawkw** [comment]
My bad --- there should have been a test for this. Will fix it tomorrow.

**Comment by hawkw** [comment]
Ah, okay, looks like the reason this wasn't caught is because all of the unit tests for proxy metrics only test the inbound side, and the doubled comma bug only occurred for metrics collected on the outbound. Will rectify the lack of tests as well.

**Comment by hawkw** [comment]
PR for a fix is up: #601
